study/cyber-tutor/quiz/js/app.js

```diff
@@ -0,0 +1,97 @@
+const myQuestions = [
+  {
+    question: "What does cybersecurity protect?",
+    answers: {
+      a: "Information systems, programs, and data ",
+      b: " Houses and neighborhoods ",
+      c: " Jobs"
+    },
+    correctAnswer: "a"
+  },
+  {
+    question: "What does a hacker use to gain access to a system?",
+    answers: {
+      a: "Paper and pencil ",
+      b: " Computer ",
+      c: " Hammer"
+    },
+    correctAnswer: "b"
+  },
+];
+function createQuiz(){
```

Comment:
Add a new line between declarations of functions or variables and the next block of code, just to make it a little more readable